turbo.json

```diff
@@ -136,84 +136,79 @@
       "outputs": []
     },
     "//#test:integration:ap-flows": {
-      "dependsOn": ["^@clerk/clerk-js#build", "^@clerk/backend#build", "^@clerk/nextjs#build"],
+      "dependsOn": ["@clerk/clerk-js#build", "@clerk/backend#build", "@clerk/nextjs#build"],
```

Comment:
All of the adjustments here remove the `^` for each dependency. The `^` says to wait on the package's dependencies, which for a root task like this effectively seems like a no-op.

I've validated that this properly caches the integration test run only if the dependent build steps remain unchanged, which is the desired behavior.